<launch>

  <!-- We assume that the images (depth and rgb) are already aligned default is kinect camera topics-->
  <arg name="rgbd_topic_name" default="rgbd_image" />
  <arg name="depth_camera_info_topic" default="/camera/depth_registered/camera_info" />
  <arg name="depth_image_topic" default="/camera/depth_registered/image_raw" />
  <arg name="rgb_camera_info_topic" default="/camera/rgb/camera_info" />
  <arg name="rgb_image_topic" default="/camera/rgb/image_raw" />

  <include file="$(find freenect_launch)/launch/examples/freenect-registered-xyzrgb.launch"/>

  <include file="$(find rgbd_camera)/launch/rgbd_camera.launch" pass_all_args="true" />

</launch>